lib/server.ts

```diff
@@ -1,12 +1,43 @@
-import ws from "ws";
+import { URL } from "url";
+import ws, { ServerOptions } from "ws";
 import type WebSocket from "ws";
 import type { Driver } from "zwave-js";
 import { EventForwarder } from "./forward";
 import type * as OutgoingMessages from "./outgoing_message";
 import { IncomingMessage } from "./incoming_message";
 import { dumpState } from "./state";
+import { Server as HttpServer, ServerOptions as HttpServerOptions, createServer } from 'http'
+import { once } from 'events'
 import { version } from "./const";
 
+interface ZwavejsServerOptions {
+  port: number
+}
+
+export class ZwavejsServer {
+  private server: HttpServer;
+  private wsServer: ws.Server;
+  private sockets: Clients;
+
+  constructor(private driver: Driver, private options: ZwavejsServerOptions) {}
+
+  async start() {
+    this.server = createServer()
+    this.wsServer = new ws.Server({ server: this.server });
```

Comment:
On what port are we listening for incoming connections now? Just any path ? 

We should update the README.

Reply:
The port of the server, 3000

Reply:
WebSockets are based on the HTTP spec, and so will always rely on full URLs. 

I just checked and with this approach, any path works. So it's backwards compatible but we should still update the README to clarify that they should just connect to `/`.

Reply:
We should also update `cli/client.ts` to no longer use `/zjs`

Reply:
ok doing it now

Reply:
done